browser/lib/transport/jsonptransport.js

```diff
@@ -94,7 +105,7 @@ var JSONPTransport = (function() {
 			params = this.params,
 			self = this;
 
-		params.callback = 'Ably._._(' + id + ')';
+		params.callback = jsonpcb + '._(' + id + ')';
```

Comment:
Nice.  I had a niggling feeling the JSONP needed to be treated differently, but in the end skipped over it :/